openfecwebapp/templates/legal-search-results-advisory_opinions.html

```diff
@@ -1,45 +1,55 @@
-  [{'value' : 'F', 'text': 'Final Opinion'},
-   {'value' : 'V', 'text': 'Votes'},
-   {'value' : 'D', 'text': 'Draft Documents'},
-   {'value' : 'R', 'text': 'AO Request, Supplemental Material, and Extensions of Time'},
-   {'value' : 'W', 'text': 'Withdrawal of Request'},
-   {'value' : 'C', 'text': 'Comments and Ex parte Communications'},
-   {'value' : 'S', 'text': 'Commissioner Statements'}],
-      ao_category or ['F']) }}
-  {{ legal.checkbox_filter('ao_is_pending', 'Show only pending requests', ao_is_pending)}}
-  {{ legal.text_filter('ao_requestor', 'Requestor Name', ao_requestor) }}
-  {{ legal.dropdown('ao_requestor_type', 'Requestor Type',
-  [ {'value': 0, 'text': 'Any'},
-    {'value': 1, 'text': 'Federal candidate/candidate committee/officeholder'},
-    {'value': 2, 'text': 'Publicly funded candidates/committees'},
-    {'value': 3, 'text': 'Party committee, national'},
-    {'value': 4, 'text': 'Party committee, state or local'},
-    {'value': 5, 'text': 'Nonconnected political committee'},
-    {'value': 6, 'text': 'Separate segregated fund'},
-    {'value': 7, 'text': 'Labor Organization'},
-    {'value': 8, 'text': 'Trade Association'},
-    {'value': 9, 'text': 'Membership Organization, Cooperative, Corporation W/O Capital Stock'},
-    {'value': 10, 'text': 'Corporation (including LLCs electing corporate status)'},
-    {'value': 11, 'text': 'Partnership (including LLCs electing partnership status)'},
-    {'value': 12, 'text': 'Governmental entity'},
-    {'value': 13, 'text': 'Research/Public Interest/Educational Institution'},
-    {'value': 14, 'text': 'Law Firm'},
-    {'value': 15, 'text': 'Individual'},
-    {'value': 16, 'text': 'Other'}],
-  ao_requestor_type) }}
+<div class="js-accordion accordion--neutral" data-content-prefix="filter" data-open-first="true">
+  <button type="button" class="js-accordion-trigger accordion__button">AO Information</button>
+  <div class="accordion__content">
+    {{ text.field('search', 'Keywords', value=query)}}
+    {{ checkbox.checkbox_dropdown(
+      'ao_cateogyr',
+      'Document type',
+      selected=constants.ao_doc_types,
+      options=constants.ao_doc_types_extended,
+    ) }}
+  </div>
+  <button type="button" class="js-accordion-trigger accordion__button">AO Information</button>
+  <div class="accordion__content">
+    {{ text.field('ao_name', 'AO name', value=ao_name)}}
+    {{ text.field('ao_no', 'AO number', value=ao_no)}}
+    {{ legal.date_filter('ao_min_date', 'ao_max_date', 'Issued date', ao_min_date, ao_max_date)}}
+    {{ legal.checkbox_filter('ao_is_pending', 'Show only pending requests', ao_is_pending)}}
+  </div>
+  <button type="button" class="js-accordion-trigger accordion__button">Requestor</button>
+  <div class="accordion__content">
+    {{ text.field('ao_requestor', 'Requestor Name', value=ao_requestor)}}
+    {{ legal.dropdown('ao_requestor_type', 'Requestor Type',
+    [ {'value': 0, 'text': 'Any'},
```

Comment:
Can this be refactored to use the constant defined in Python? Just noticing that we've got this data structure in the code twice.